From 1861feeee393be8996732f43d92dfa7377671741 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Wed, 14 Mar 2018 16:06:46 +0800
Subject: [PATCH 3561/5242] MLK-17800-1: ARM64: dts: Remove the spdif
 workaround for imx8qxp B0

commit  3beea03235f643ede0cfbf6a00f460d98881671d from
https://source.codeaurora.org/external/imx/linux-imx.git

Fixes commit e76fb2786d32 ("MLK-15937-4: ASoC: fsl_spdif: Use
DMA workaround for SPDIF")

The hardware issue that SPDIF dma event is not connected to
EDMA correctly is fixed in B0.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Reviewed-by: Viorel Suman <viorel.suman@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../freescale/fsl-imx8qxp-lpddr4-arm2-spdif.dts    |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-spdif.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-spdif.dts
index f333c2b..c662262 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-spdif.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-spdif.dts
@@ -48,13 +48,12 @@
 };
 
 &spdif0 {
-	compatible = "fsl,imx8qxp-v1-spdif";
+	compatible = "fsl,imx8qm-spdif";
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_spdif0>;
 	assigned-clocks = <&clk IMX8QXP_AUD_PLL0_DIV>,
 			<&clk IMX8QXP_AUD_ACM_AUD_PLL_CLK0_DIV>,
 			<&clk IMX8QXP_AUD_ACM_AUD_REC_CLK0_DIV>;
 	assigned-clock-rates = <786432000>, <49152000>, <24576000>;
-	dmas = <&edma2 23 0 7>, <&edma2 21 0 6>;
 	status = "okay";
 };
-- 
1.7.9.5

